### PR TITLE
S3 access guide

### PR DESCRIPTION
**Pull request by abuiquoc**
guide to access EO data via S3

### CODE REVIEW

**Review by jdries — CHANGES_REQUESTED** [review]
Preview here: https://eu-cdse.github.io/documentation/APIs/S3.html
You will also want to add this new page to the navigation config:
https://github.com/eu-cdse/documentation/blob/publish/_quarto.yml

File: APIs/S3.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+In order to obtain secrets, create a ticket to CDSE support. Our operators will provide you with the appropriate credentials. You must provide in such request the e-mail address of the registered CDSE user for whom the credentials will be generated.
+
+## Example access using s3cmd
+
+Having the access and secret key together with the endpoint, you can use any tool to handle access via S3. Below is an example of how to access EO Data using the s3cmd.
+
+First, we recommend to create a configuration file. You can create it with tools like vi/vim or nano:
+```
+vi .s3cfg
```
[the remark this answers is not in the document]
Comment (abuiquoc) [reply APIs/S3.md:17]: This example assumes use of a Linux environment. Commands does not necessarily have to be executed from a virtual machine, client can access VM that is created on a non-Linux image.

File: APIs/S3.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+To generate the necessary credentials you must have a registered account on dataspace.copernicus.eu. If you don't have an account, you can register [here](https://identity.dataspace.copernicus.eu/auth/realms/CDSE/login-actions/registration?client_id=cdse-public&tab_id=y6n555-H1GU).
+
+## Request for secrets
+
+In order to obtain secrets, create a ticket to CDSE support. Our operators will provide you with the appropriate credentials. You must provide in such request the e-mail address of the registered CDSE user for whom the credentials will be generated.
+
+## Example access using s3cmd
+
+Having the access and secret key together with the endpoint, you can use any tool to handle access via S3. Below is an example of how to access EO Data using the s3cmd.
```
[the remark this answers is not in the document]
Comment (abuiquoc) [reply APIs/S3.md:13]: Endpoint will be always the same, we can put "s3.dataspace.copernicus.eu" in this paragraph.

File: APIs/S3.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Accces to EO data via S3
```
[the remark this answers is not in the document]
Comment (abuiquoc) [reply APIs/S3.md:1]: _Should we write that this is the preferred and most performant access method?_
It depends on the end user workflow.

 _Are there costs involved for remote access?_
Access to S3 can be requested by any end user for free, however, access (download speed and requests) in the future will be limited, depending on the type of user

[the remark this answers is not in the document]
Comment (jdries) [reply APIs/S3.md:1]: ok, please add this info. Elaborate a bit on performance and different user workflows?

File: APIs/S3.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jdries) [reply APIs/S3.md:13]: ok, please add endpoint somewhere at the top

File: APIs/S3.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jdries) [reply APIs/S3.md:17]: ok, please clarify that linux is assumed here, maybe add a link to the tool that you use here

**Comment by abuiquoc** [comment]
@jdries changes has been added

**Comment by jdries** [comment]
@abuiquoc  @Pratichhya  the content looks fine, but now the branch has a merge conflict. There's a button to resolve this, can be merged afterwards.

**Comment by Pratichhya** [comment]
@abuiquoc, Sorry for the delay, it has been merged to the publish branch and will soon reflect on the website.

**Pull request merged**